test/overlay.html

```diff
@@ -411,54 +424,55 @@
     describe('revealDate', () => {
       let overlay, monthScroller;
 
-      function fixtureOverlayContent({monthScrollerItems, monthScrollerOffset}, done) {
-        overlay = fixture('overlay');
-        monthScroller = overlay.$.monthScroller;
-        monthScroller.style.setProperty('--vaadin-infinite-scroller-buffer-offset', monthScrollerOffset);
-        monthScroller.updateStyles({'--vaadin-infinite-scroller-buffer-offset': monthScrollerOffset});
-        monthScroller.style.height = `${270 * monthScrollerItems}px`;
-        overlay.i18n = getDefaultI18n();
-        overlay.$.monthScroller.bufferSize = 3;
-        overlay.$.yearScroller.bufferSize = 3;
-        overlay.initialPosition = new Date(2021, 1, 1);
-        setTimeout(() => {
-          Polymer.RenderStatus.afterNextRender(overlay, done);
-        }, 1);
-      }
-
       describe('height(visible area) < height(item)', () => {
         beforeEach((done) => {
-          fixtureOverlayContent({
+          overlay = customizeFixture({
+            initialPosition: new Date(2021, 1, 1),
             monthScrollerItems: 0.5,
             monthScrollerOffset: 0
-          }, done);
-        });
-
-        it('should scroll when the month is above the visible area', () => {
```

Comment:
The old test cases for `height(visible area) < height(item)` are obsolete with the new mode.